src/app/clusters/ias-zone-server/ias-zone-server.cpp

```diff
@@ -249,10 +249,11 @@ MatterIasZoneClusterServerPreAttributeChangedCallback(const app::ConcreteAttribu
     // currently existing in the field.
     bindingEntry.type      = EMBER_UNICAST_BINDING;
     bindingEntry.local     = endpoint;
-    bindingEntry.clusterId = ZCL_IAS_ZONE_CLUSTER_ID;
+    bindingEntry.clusterId = Optional<ClusterId>(ZCL_IAS_ZONE_CLUSTER_ID);
     bindingEntry.remote    = emberAfCurrentCommand()->apsFrame->sourceEndpoint;
     bindingEntry.nodeId    = destNodeId;
 
+    bool foundSameEntry = false;
```

Comment:
Why are we changing this code in any way other than what is necessary to make it compile?